src/Core/Settings.h

```diff
@@ -779,6 +779,7 @@ static constexpr UInt64 operator""_GiB(unsigned long long value)
     M(ExecuteMode, exec_mode, ExecuteMode::NORMAL, "Control query execute mode", 0) \
     M(UInt64, checkpoint_interval, 0, "Checkpoint interval in seconds", 0) \
     M(UInt64, javascript_uda_max_concurrency, 1, "Control the concurrency of JavaScript UDA in a query", 0) \
+    M(Float, replay_speed, 0., "Control the replay speed", 0) \
```

Comment:
Need more detailed description:
The replay_speed controls the rate at which data is played back (based on the actual inserted interval)
1) `replay_speed <= 0`, means not replay
2) `0 < replay_speed < 1`, means replay slower
3) `replay_speed == 1`, means replay by actual ingest interval
4) `1 < replay_speed < <max_limit>`, means replay faster